src/Category/Algebra.agda

```diff
@@ -0,0 +1,119 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Algebras for endofunctors
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+open import Category.Category using (Category)
+
+module Category.Algebra {ℓ} (cat : Category ℓ) where
+
+open import Level
+open Category cat
+open import Category.Functor cat cat
+  renaming (RawFunctor to RawEndoFunctor)
+
+module _ {fun : Obj → Obj} (F : RawEndoFunctor fun) where
+  open RawEndoFunctor F
+  
+  record Algebra : Set (suc ℓ) where
+    field
+      Carrier  : Obj
+      evaluate : fun Carrier ⇒ Carrier
+  open Algebra public
+
+  open import Category.Structures
+  module AlgebraCategory (isCat : IsCategory cat) (isFun : IsFunctor cat cat F) where
+    open IsCategory isCat
+    open IsFunctor isFun
+    open ≈-Reasoning
+
+    record AlgebraMorphism (algu : Algebra) (algv : Algebra) : Set ℓ where
```

Comment:
Given that this definition is inside a module named `Category.Algebra` it probably doesn't need `Algebra` in the name and should just be called `Morphism`.